packages/create/src/cli.ts

```diff
@@ -25,17 +27,48 @@ program
         message: 'Which template would you like to use?',
         choices: [
           { title: 'webpack', value: 'webpack' },
-          { title: 'vite', value: 'vite' },
+          { title: 'vite (experimental)', value: 'vite' },
         ],
         initial: 0,
       });
 
       selectedTemplate = template;
     }
 
-    console.log('🚧 create-sku is not yet implemented');
-    console.log(`Project name: ${projectName}`);
-    console.log(`Template: ${selectedTemplate}`);
+    if (selectedTemplate === 'vite') {
+      const { confirmVite } = await prompts({
+        type: 'confirm',
+        name: 'confirmVite',
+        message:
+          '⚠️  Vite bundler in sku is currently experimental and not yet suitable for production use. Continue?',
+        initial: false,
+      });
+
+      if (!confirmVite) {
+        console.log(
+          '❌ Cancelled. Use `webpack` template for a stable production-ready experience.',
+        );
+        process.exit(0);
```

Comment:
I feel like anything other than a success should be exit code 1 to signify some kind of failure.